input/vocabulary/valueset-provider-taxonomy.json

```diff
@@ -0,0 +1,3534 @@
+{
+  "resourceType": "ValueSet",
+  "id": "provider-taxonomy",
+  "meta": {
+      "lastUpdated": "2023-03-26T15:21:02.749+11:00",
+      "profile": [
+          "http://hl7.org/fhir/StructureDefinition/shareablevalueset"
+      ]
+  },
+  "extension": [
+      {
+          "url": "http://hl7.org/fhir/StructureDefinition/structuredefinition-wg",
+          "valueCode": "vocab"
+      }
+  ],
+  "url": "http://hl7.org/fhir/ValueSet/provider-taxonomy",
+  "identifier": [
+      {
+          "system": "urn:ietf:rfc:3986",
+          "value": "urn:oid:2.16.840.1.113883.4.642.3.51"
+      }
+  ],
+  "version": "5.0.0",
+  "name": "ProviderTaxonomyCodes",
+  "title": "Provider Taxonomy Codes",
+  "status": "draft",
+  "experimental": true,
+  "date": "2023-03-26T15:21:02+11:00",
+  "publisher": "FHIR Project team",
+  "contact": [
+      {
+          "telecom": [
+              {
+                  "system": "url",
+                  "value": "http://hl7.org/fhir"
+              }
+          ]
+      }
+  ],
+  "description": "NUCC Healthcare Provider Taxonomy codes",
+  "jurisdiction": [
+      {
+          "coding": [
+              {
+                  "system": "http://unstats.un.org/unsd/methods/m49/m49.htm",
+                  "code": "001",
+                  "display": "World"
```

Comment:
display = "United States of America"

Reply:
The 'display ' is updated as "United States of America".

Kindly review.